internal/python/plan.go

```diff
@@ -535,6 +558,9 @@ func determineStartCmd(ctx *pythonPlanContext) string {
 
 		if framework == types.PythonFrameworkFastapi {
 			commandSegment = append(commandSegment, "uvicorn", wsgi, "--host 0.0.0.0", "--port "+wsgilistenedPort)
+		}
+		if framework == types.PythonFrameworkSanic {
```

Comment:
Use `else if` is better.

```suggestion
		else if framework == types.PythonFrameworkSanic {
```